client/card-readers/settings/sections/branding-details.js

```diff
@@ -4,42 +4,21 @@
  */
 import React from 'react';
 import { __ } from '@wordpress/i18n';
-import { ColorPaletteControl } from '@wordpress/block-editor';
 import { useEffect } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import BrandingFileUpload from '../file-upload';
 
-import {
-	useAccountBrandingPrimaryColor,
```

Comment:
We should probably also get rid of these hooks in `client/data/settings/hooks.js`.